scripts/analysis/compare_terrain_loss.py

```
@@ -0,0 +1,190 @@
+
+import json
+import sys
+from typing import Dict, List, Any
+
+
+def load_json_file(filepath: str) -> List[Dict[str, Any]]:
+    """Load JSON file and return data."""
+    try:
+        with open(filepath, "r", encoding="utf-8") as f:
+            return json.load(f)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)